### PR TITLE
Fix mapped injectors

### LINKED ISSUE #9542: dna injectors and  vaults
(WEB REPORT BY: pvak REMOTE: 172.93.109.202:7777)

> Revision (Should be above if you're viewing this from ingame!)
> 
> General description of the issue
> the dna injectors that spawn in one of the vaults dont work
> 
> What you expected to happen
> to work
> 
> What actually happened

they dont

> Steps to reproduce if possible

hit yourself with the dna injector

### PR DESCRIPTION
**Pull request by clusterfack**
Moved genetics earlier in startup due to it not having dependencies
Fixes #9542

### CODE REVIEW

File: code/game/objects/items/weapons/dna_injector.dm [hunk 1]
```diff
@@ -21,6 +21,10 @@
 /obj/item/weapon/dnainjector/New()
 	. = ..()
 
+	if(ticker)
+		initialize()
```
Comment (Intigracy) [inline code/game/objects/items/weapons/dna_injector.dm:25]: If the ticker check fails, won't this just never initialize?

**Comment by clusterfack** [comment]
I wanted to sarcastically ask you *what do you think the ticker does* but I'll just tell you, it initializes all objects

**Comment by Intigracy** [comment]
Oh.

TMYK

**Comment by clusterfack** [comment]
Among other things, you can see it's real purpose in the game ticker.dm modified here

**Comment by PJB3005** [comment]
@clusterfack no actually that's false.

The ticker doesn't initialize them, the master controller does.

**Pull request merged**